tests/python/tank_test/tank_test_base.py

```diff
@@ -31,7 +31,43 @@
 TANK_TEMP = None
 TANK_SOURCE_PATH = None
 
-__all__ = ['setUpModule', 'TankTestBase', 'tank']
+__all__ = ['setUpModule', 'TankTestBase', 'tank', 'interactive', 'skip_if_pyside_missing']
+
+
+def interactive(func):
+    """
+    Decorator that allows to skip a test if the interactive flag is not set
+    on the command line.
+    :param func: Function to be decorated.
+    :returns: The decorated function.
+    """
+    interactive_in_argv = "--interactive" not in sys.argv
+    return unittest.skipIf(
+        interactive_in_argv,
+        "add --interactive on the command line to run this test."
+    )(func)
+
+
+def _is_pyside_missing():
+    """
+    Tests is PySide is available.
+    :returns: True is PySide is available, False otherwise.
+    """
+    try:
+        import PySide
+        return False
+    except ImportError:
```

Comment:
I think we should print out a message if QT is being skipped. "test x is not running because PySide could not be imported". Does that make sense?